src/constants.ts

```diff
@@ -9,3 +9,8 @@ export const DEFAULT_AUTHORIZE_TIMEOUT_IN_SECONDS = 60;
 export const DEFAULT_POPUP_CONFIG_OPTIONS: PopupConfigOptions = {
   timeoutInSeconds: DEFAULT_AUTHORIZE_TIMEOUT_IN_SECONDS
 };
+
+/**
+ * @ignore
+ */
+export const DEFAULT_SILENT_TOKEN_RETRY_COUNT = 5;
```

Comment:
This doesn't look like it's configurable ... is it? Shouldn't it be? Also, default 5 times seems like a lot ...

Reply:
I started down the road of making it configurable but ultimately it feels like as a first pass it would be better to set this to something and see what feedback we get around it. It might be what we have right now (a sensible default) is enough for 99% of cases.

I'm not precious about the 5, but if you feel it's too high, the only other number I suggest would be 3. WYDT?

Reply:
As long as you think this has been built in a way to make adding that config easily, then I'm 👍 

Given that the timeout is 60 seconds (😮), 5 retries is potentially ... 6 minutes?? Is the UI frozen for that time? For network failures I feel like 1, maybe 2 is sufficient.

Reply:
Ah, so the timeout you're referring to is our own timeout for the `/authorize` call when using an iframe. You raise a good point though, I'm not sure what the default timeout for `fetch` is when it comes to network errors - let me check that out first.

Reply:
To circle back on this, `fetch` itself doesn't prescribe any time out at all. So it will essentially wait until something fails before returning.

To combat this, I will wrap the fetch call and the retries in a wrapper than enforces a timeout, say, 10 seconds, and lower the retry count to 3. How does that sound?

Reply:
Sounds much better!